quirks/shared-credentials.json

```diff
@@ -446,7 +446,13 @@
     },
     {
         "shared": [
-            "ting.com", 
+            "telegram.org",
+            "telegram.me"
+        ]
```

Comment:
From what I can tell, telegram.me redirects to telegram.org, so a “from” / “to” relationship would be more correct here.

Reply:
It does redirect when opened in the browser, however Password Autofill on iOS suggests an account for login to the Telegram app only if "telegram.me" is also added to the domains in the Passwords app.

Would "from" or "to" relationship work in this case?

Reply:
On iOS, we will interpret this correctly and do exactly what you want. :)

Reply:
I changed the relationship type in the new commit. Does that look right?